### PR TITLE
Add `icp.international`, `public.icp.international` and `verified.icp.international`

### PR DESCRIPTION
**Pull request by essesoul**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [ ] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

International Internet Content Provider Verification (IICPV) is a public welfare group, we hope to provide a platform for friends (whether professional or non-professional) who create websites all over the world to communicate and exchange friendship links. We hope that more excellent websites created by individuals can be seen by the public.

Organization Website:  [https://www.iicpv.org/](https://www.iicpv.org)

Reason for PSL Inclusion
====

Website creators can obtain a custom like `example.icp.international` domain name for testing pages

- Domain abuse - while we actively find and shut down spam/phishing subdomains, some spam check sites flag the entire domain
- Analytics pollution - Google Analytics and FB pixel cookie integrations result in analytics cookies getting set at domain level
- Site Settings - without PSL, site settings changes in the browser apply for all customer sites at the same time

Number of users this request is being made to serve:

It is estimated that there will be more than 10,000 icp.lol subdomains, more than 1,000 icp.international subdomains of icp.international, more than 10,000 public.icp.international test subdomains, and more than 10,000 verified.icp.international subdomains

DNS Verification via dig
=======

```
dig +short TXT _psl.icp.international
"https://github.com/publicsuffix/list/pull/1751"
```

```
dig +short TXT _psl.public.icp.international
"https://github.com/publicsuffix/list/pull/1751"
```

```
dig +short TXT _psl.verified.icp.international
"https://github.com/publicsuffix/list/pull/1751"
```

Results of Syntax Checker (`make test`)
=========

All tests pass

### CODE REVIEW

**Comment by simon-friedberger** [comment]
Can you please provide updated user numbers?

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12132,6 +12132,12 @@ to.leg.br
 // Submitted by Wolfgang Schwarz <admin@intermetrics.de>
 pixolino.com
 
+// International Internet Content Provider Verification
+// Submitted by Gary Chan <gary@iicpv.org>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:12136]: Please use a non-personal e-mail.

**Comment by simon-friedberger** [comment]
* [ ] Expiration (Note: Must STAY >2y at all times)
  * [ ] icp.international expires 2024-05-01
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [x] Sorting
* [ ] Reasoning/Organization description

**Comment by simon-friedberger** [comment]
Please provide updated user numbers. The description also refers to `icp.lol` which in not actually in the PR. Given the name, do you do any actual verification?

**Comment by essesoul** [comment]
icp.lol is no longer planned to be a public suffix, so I removed it

**Comment by essesoul** [comment]
> Please provide updated user numbers. 
about 3,000

**Pull request closed without merging**